composer.json

```
@@ -7,15 +7,17 @@
             "email" : "olivier.mansour@m6.fr"
         }
     ],
+    "config": {
+        "bin-dir": "bin/"
+    },
     "require" : {
-        "php" : "^7.0"
+        "php" : "^7.4|^8.0",
+        "symfony/symfony": "^2.7|^3.0|^4.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "^2.8|^3.0",
+        "atoum/atoum" : "^3.0|^4.0",
```

```suggestion
        "atoum/atoum" : "^4.0",
```

no need to support an older version of atoum for php 7.4+